runtime/lua/vim/lsp/rpc.lua

```diff
@@ -18,7 +18,7 @@ end
 --- Embeds the given string into a table and correctly computes `Content-Length`.
 ---
 ---@param encoded_message (string)
----@returns (table) table containing encoded message and `Content-Length` attribute
+---@return string containing encoded message and `Content-Length` attribute
```

Comment:
The type isn't part of the sentence.
```suggestion
---@return string Encoded message and `Content-Length` attribute
```

Reply:
obviously wrong . 
```lua
local function format_message_with_content_length(encoded_message)
  return table.concat({
    'Content-Length: ',
    tostring(#encoded_message),
    '\r\n\r\n',
    encoded_message,
  })
end
```